scripts/assertions.bash

```diff
@@ -51,3 +51,15 @@ assert_tarball_not_contains_tags() { TARBALL="$1"; shift
 	tarball_tag_check_prep "$TARBALL" "$@"
 	assert_tags_do_not_exist_locally "$@"
 }
+
+assert_is_binfmt_registered() { 
+  [ -f "/proc/sys/fs/binfmt_misc/$1" ] && return 0
+  return 1
+}
```

Comment:
I like the explicitness of this, but I think BATS automatically has `set -e` enabled so the returns may not be necessary? Happy either way though, and I could be wrong, Bash is anything but uniform and predictable in behaviour sometimes.